src/components/pages/Onboarding/OnboardingCamera.tsx

```diff
@@ -23,9 +23,8 @@ export class OnboardingCamera extends React.Component<OnboardingCameraProps> {
       <React.Fragment>
         <Text style={sharedStyles.paragraph}>
           {this.props.inviterFullName
-            ? "Please record a video of yourself stating your identity to accept your invite from " +
-              this.props.inviterFullName
-            : "Please record a video of yourself stating your identity."}
+            ? `Create your public profile's identity video to accept your invite from ${this.props.inviterFullName}!`
```

Comment:
I find this phrasing a bit awkward --
how about "Accept your invite from {} by taking a video of yourself stating your name for your public profile!"
and "Take a video of yourself stating your name for your public profile!" And maybe some context, "This video is how other people know who you are."